exporters/otlp/recordable_test.cc

```diff
@@ -0,0 +1,64 @@
+#include "exporters/otlp/recordable.h"
+
+#include <gtest/gtest.h>
+
+using namespace opentelemetry::exporter::otlp;
+
+TEST(Recordable, SetIds)
+{
+    Recordable rec;
+
+    const uint8_t trace_id_buff[16] = {0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1};
+    const uint8_t span_id_buff[8] = {0,0,0,0,0,0,0,2};
+    const uint8_t parent_span_id_buff[8] = {0,0,0,0,0,0,0,3};
+
+    opentelemetry::nostd::span<const uint8_t, 16>trace_id_span(trace_id_buff, 16);
```

Comment:
Please run format. How come CI didn't catch this?

Reply:
Thanks! I'm not sure why CI didn't catch this... we're using `clang-format version 8.0.0-3~ubuntu18.04.2`, perhaps this check is not part of that?